src/zoid/buttons/util.js

```diff
@@ -50,8 +82,8 @@ export function createVenmoExperiment() : Experiment | void {
     const isEligibleForVenmo = fundingEligibility && fundingEligibility[FUNDING.VENMO] && fundingEligibility[FUNDING.VENMO].eligible;
 
     // exclude buyers who are not eligible
-    // exclude integrations using enable-funding=venmo
-    if (!isEligibleForVenmo || isEnableFundingVenmo) {
+    // exclude non-desktop integrations using enable-funding=venmo
+    if (!isEligibleForVenmo || (isEnableFundingVenmo && isDevice())) {
```

Comment:
Same here with leveraging `supportsQRPay`
```suggestion
    if (!isEligibleForVenmo || (isEnableFundingVenmo && supportsQRPay(FUNDING.VENMO) === false)) {
```